inst/shiny/plotBoxplots.R

```diff
@@ -0,0 +1,84 @@
+plotBoxplots <- reactive({
+  
+  w.use <- w.use()
+  
+  data.elements <- c(df[["data.element"]],df[["data.element.y"]])
+  
+  areas.p2e <- df[["area"]]
+  
+  if(all(df[["areas"]] %in% areas.p2e)){
+    areas.p2e <- NA
+  }
+  notch <- input$notchOn
+  yrs <- NA # may want to code ability to select certain years
```

Comment:
I think this is fine. I've made a few decisions like this where the options on the app are a little more simplified than the package function. If you offer too many choices, the UI starts to get confusing.